lab_1_keywords_tfidf/main.py

```diff
@@ -48,7 +66,19 @@ def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not len(tokens) != 0 and not isinstance(tokens, list):
```

Comment:
`len(tokens) != 0` is a bad check 
consider using `if tokens`

Reply:
but how will it understand that I mean len?

Reply:
good question!
in python, empty objects are considered to be equal to False